code/modules/power/power.dm

```diff
@@ -1,26 +1,37 @@
+//////////////////////////////
+// POWER MACHINERY BASE CLASS
+//////////////////////////////
+
+/////////////////////////////
+// Definitions
+/////////////////////////////
+
 /obj/machinery/power
 	name = null
 	icon = 'icons/obj/power.dmi'
 	anchored = 1.0
 	var/datum/powernet/powernet = null
-	var/directwired = 1		// by default, power machines are connected by a cable in a neighbouring turf
-							// if set to 0, requires a 0-X cable on this turf
 	use_power = 0
 	idle_power_usage = 0
 	active_power_usage = 0
 
-/obj/machinery/power/Destroy()
+/obj/machinery/power/Del()

```

Comment:
No point to this, /datum/Del will always call Destroy(), and ideally, everything should be qdelled anyways, therefore calling Destroy first.